### DISCUSSION TITLE
New pull with challenge completed

### DISCUSSION THREAD

**Discussion by mariamor99** [TheAlbertDev/MASB#48] [Q&A] [answered]
Hi Albert! I corrected the challenge of the lab session P1-A but I can't make a new pull request.
Can you commit a new version of my branches A and B so I can create a new pull request?
Thanks!

**Comment by Albert-Alvarez** [comment, marked as answer]
Hi @mariamor99,
Once created, a Pull Request already updated itself with last commits from the branch to be merged. So, if you have applied the changes in the corresponding branch, the Pull Request is already updated and I can see the new changes 😉

For example, you Pull Request for your branch A, now looks like this:

![image](https://user-images.githubusercontent.com/34403794/110236178-a8166900-7f34-11eb-90ba-228988aa507b.png)
